file_catalog/server.py

```diff
@@ -408,7 +408,10 @@ def get(self):
             logging.warn('query parameter error', exc_info=True)
             self.send_error(400, message='invalid query parameters')
             return
-        files = yield self.db.find_files(**kwargs)
+        if 'count' in kwargs:
+            files = yield self.db.count_files(**kwargs)
```

Comment:
I don't think that dynamically changing the type of `files` is intuitive. I expected `files` to always be a list.